src/lib/Form/Data/Policy/PolicyUpdateData.php

```diff
@@ -23,9 +23,11 @@ class PolicyUpdateData
 
     public function __construct(?Policy $policy = null)
     {
-        $this->module = $policy->module;
-        $this->function = $policy->function;
-        $this->limitations = $policy->limitations;
+        if($policy instanceof Policy){
```

Comment:
Code style space after `if` statement

Reply:
We tend to use early returns and also I'd check if it's null instead an instance of class:
`if (null === $policy)`